applications/ParticleMechanicsApplication/custom_utilities/mpm_particle_generator_utility.h

```diff
@@ -55,16 +55,13 @@ namespace MPMParticleGeneratorUtility
     /// Get integration weights of the geometry for the given integration method
     void GetIntegrationPointVolumes(const GeometryType& rGeom, const IntegrationMethod IntegrationMethod, Vector& rIntVolumes);
 
-    /// Get integration weights of the geometry for the given integration method
-    void GetIntegrationPointArea(const GeometryType& rGeom, const IntegrationMethod IntegrationMethod, Vector& rIntVolumes);
-
     /// Get integration method and shape function values for the given element
     void DetermineIntegrationMethodAndShapeFunctionValues(const GeometryType& rGeom, const SizeType ParticlesPerElement,
         IntegrationMethod& rIntegrationMethod, Matrix& rN, bool& IsEqualVolumes);
 
-    /// Get integration method and shape function values for the given condition
-    void DetermineConditionIntegrationMethodAndShapeFunctionValues(const GeometryType& rGeom, const SizeType ParticlesPerElement,
-        IntegrationMethod& rIntegrationMethod, Matrix& rN, bool& IsEqualVolumes);
+    /// Get integration method and shape function values for the given element
+    void DetermineGeometryIntegrationMethod(const GeometryType& rGeom, const SizeType ParticlesPerElement,
+        IndexType& rNumPointsPerSpan);
```

Comment:
This is only used for conditions, not for elements. I think you should rename `ParticlesPerElement` with `ParticlesPerCondition`

Reply:
Thanks for the suggestion - I'll rename it